elastalert/alerts.py

```diff
@@ -1295,3 +1295,36 @@ def alert(self, matches):
     def get_info(self):
         return {'type': 'simple',
                 'simple_webhook_url': self.simple_webhook_url}
+
+
+class MSTeamsAlerter(Alerter):
+    def __init__(self, rule):
+        super(MSTeamsAlerter, self).__init__(rule)
+        msteams_webhook_url = self.rule.get('msteams_webhook_url')
+        if isinstance(msteams_webhook_url, basestring):
+            msteams_webhook_url = [msteams_webhook_url]
+        self.msteams_webhook_url = msteams_webhook_url
+        self.msteams_proxy = self.rule.get('msteams_proxy')
+
+
+    def alert(self, matches):
+        payload = {
+            'isMultiline': True,
+            'text': '\n'.join(['```{}```'.format(match) for match in matches])
```

Comment:
Almost every other alerter uses ``BasicMatchString(self.rule, match)`` to generate the alert text. It's fine to have the raw matches too, but I'd like to see that behind a config setting.

I know the other alerters are very inconsistent, but those need to be fixed, not copied.